ui/src/data-services/models/occurrence-details.ts

```diff
@@ -47,4 +110,19 @@ export class OccurrenceDetails extends Occurrence {
       }),
     }
   }
+
+  private _isIdentificationOverridden(identification: any) {
```

Comment:
@mihow here is the logic for when to display an identification as overridden. I'm not seeing withdrawn being set in BE response, which is why I also have some FE logic here. Maybe we can remove the FE stuff later here

Reply:
Oh and FIY the response I'm talking about is the occurrence details one, with the list of identifications